src/js/module.ts

```diff
@@ -162,7 +162,14 @@ function installStdlib(Module: Module, stdlibURL: string) {
  * @private
  */
 export function initializeFileSystem(Module: Module, config: ConfigType) {
-  installStdlib(Module, config.indexURL + "python_stdlib.zip");
+  let stdLibURL;
+  if (config.stdLibURL != undefined) {
+    stdLibURL = config.stdLibURL;
```

Comment:
Question: what happens if we pass relative urls?

Reply:
It just passes them to fetch, so relative URLs also work. That's what is tested currently.